### PR TITLE
[apollo-codegen] Update to latest version 0.20

### PR DESCRIPTION
**Pull request by kostspielig**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/apollographql/apollo-codegen/releases/tag/v0.20.0
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by DeTeam** [comment]
Nice 👍

File: types/apollo-codegen/index.d.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
-// Type definitions for apollo-codegen 0.16
+// Type definitions for apollo-codegen 0.20
 // Project: https://github.com/apollographql/apollo-codegen
-// Definitions by: Bradley Ayers <https://github.com/bradleyayers>
+// Definitions by: Bradley Ayers <https://github.com/bradleyayers>, Maria Carrasco <https://github.com/kostspielig>
```
Comment (bradleyayers) [inline types/apollo-codegen/index.d.ts]: Not sure if comma-separated works here, in the past I've only seen new-line-separated.

**Review by bradleyayers — APPROVED** [review]

File: types/apollo-codegen/index.d.ts (hunk 1 above)
Comment (paulvanbrenk) [reply types/apollo-codegen/index.d.ts]: comma-separated works too

Comment (bradleyayers) [reply types/apollo-codegen/index.d.ts]: Thanks!

**Pull request merged**